e2e/browser/specs/by_control.spec.js

```diff
@@ -163,5 +163,49 @@ describe("by_control", function () {
 		}));
 
 		expect(list.getAttribute("id")).toContain("ListPage1");
-	})
+	});
+
+	it("should interact with overflow toolbars", function () {
+		var presentToggleButton = element(by.control({
+			controlType: "sap.m.ToggleButton",
+			ancestor: {
+				id: "toolbar-overflow"
+			},
+			visible: false
+		}));
+		var overflowingButton = element(by.control({
+			controlType: "sap.m.Button",
+			properties: {
+				text: "Overflowing"
+			}
+		}));
+
+		presentToggleButton.isPresent().then(function (isPresent) {
+			if (isPresent) {
+				presentToggleButton.click();
+			}
+		});
+
+		// toggle button should be pressed and overflowing buttons should be shown
+		expect(overflowingButton.isPresent()).toBeTruthy();
+
+		var hiddenToggleButton = element(by.css("#toolbar-fit button[title='More']"));
+		var toolbarButton = element(by.control({
+			controlType: "sap.m.Button",
+			properties: {
+				text: "Always Visible"
+			}
+		}));
+
+		hiddenToggleButton.isPresent().then(function (isPresent) {
```

Comment:
TODO:
1. try with isDisplayed,isEnabled
2. try with executeScript and jq inside the browser => export as some helper on browser object